lib/models/command.js

```diff
@@ -197,7 +198,7 @@ let Command = CoreObject.extend({
       return this._currentTask.onInterrupt();
     } else {
       // interrupt all external commands which don't use `runTask()` with an error
-      process.exit(1);
+      return RSVP.reject();
```

Comment:
I see. Thank you for explanation. I've tried to pass an exit code but it leads to a strange result error message in the console.

I'm going to continue work here once #6779 is resolved.